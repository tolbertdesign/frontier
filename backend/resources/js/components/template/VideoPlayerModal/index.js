import VideoPlayerModal from './VideoPlayerModal.vue'

export default VideoPlayerModal
