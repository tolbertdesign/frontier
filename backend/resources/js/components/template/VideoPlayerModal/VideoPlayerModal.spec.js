import { shallowMount, createLocalVue } from '@vue/test-utils'
import VideoPlayerModal from './VideoPlayerModal.vue'

const localVue = createLocalVue()

describe('VideoPlayerModal', () => {
  it('Snapshop test for VideoPlayerModal', () => {
    const wrapper = shallowMount(VideoPlayerModal, {
      localVue,
      propsData: {
        videoId: 12345678,
      },
    })
    expect(wrapper).toMatchSnapshot()
  })
})
