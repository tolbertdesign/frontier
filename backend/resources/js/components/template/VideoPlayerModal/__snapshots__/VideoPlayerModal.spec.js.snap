// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`VideoPlayerModal Snapshop test for VideoPlayerModal 1`] = `
<div>
  <figure class="image is-16by9 mb-4"><iframe src="https://player.vimeo.com/video/12345678?autoplay=1&amp;autopause=0" frameborder="0" webkitallowfullscreen="" mozallowfullscreen="" allowfullscreen="allowfullscreen" style="position: absolute; top: 0px; left: 0px; width: 100%; height: 100%;"></iframe></figure>
  <div class="close-button z-max"></div>
</div>
`;
